Kickstarter-iOS/Views/Controllers/ProjectPageViewControllerTests.swift

```diff
@@ -590,14 +590,23 @@ internal final class ProjectPageViewControllerTests: TestCase {
       fetchProjectPamphletResult: .failure(.couldNotParseJSON)
     )
 
+    // This test was previously flakey on CI because it relied on Alamofire to download an image url to populate the user image, which may/may not be ready in time.
+    let projectWithNoUserImageURL = self.project
+      |> Project.lens.creator .~ (
+        User.template
+          |> \.avatar.large .~ ""
+          |> \.avatar.medium .~ ""
+          |> \.avatar.small .~ ""
+      )
+
```

Comment:
So the solution here is to blank out the image, right? Did you have to regenerate the existing snapshot that's used as a comparison?

Reply:
Yes the latest [commit]([9b629f3](https://github.com/kickstarter/ios-oss/pull/1700/commits/9b629f35db244a6696ad17e411fa95ceee83880d) has the newly recorded snapshots. 

When this pr was first made ready to review, the reference snapshot had the error which was a creator url, blanked out by the code above. So that is why it failed. I assumed it was recorded correctly and that was a mistake. Turns out more than one reference snapshot had the creator avatar recorded. So technically those were flakey too although for some reason this one test `testErrorFetchingProject` was the one failing the most.

The current application logic as mentioned in response to Isabel's comment is to display the creator url only if it is a valid url. We don't at the moment support an in-app image placeholder before this url is downloaded (or if its not there). That being the case, without updating in-app logic, its safest to send in a blank url string and not have the image show anything. Not perfect, but the point of these tests is to update that Pledge Container CTA not to test the creator/project images.

Anyway, tldr: yes snapshot tests needed re-recorded after nulling out images on all offending tests (3)